Packs/PrismaSaasSecurity/pack_metadata.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "Saas Security (Prisma)",
+    "name": "Saas Security (Beta)",
```

Comment:
If you want an integration to be Beta, I suggest specifying it in the integration yml file.
in the display field and using the `bet: true` property.
sse this file for example: https://github.com/demisto/content/blob/master/Packs/AWS-AccessAnalyzer/Integrations/AWS-AccessAnalyzer/AWS-AccessAnalyzer.yml

Reply:
updated pr accordingly